lib/camt_parser/053/statement.rb

```diff
@@ -39,7 +39,7 @@ def electronic_sequence_number
 
       def opening_balance
         @opening_balance ||= begin
-          bal = @xml_data.xpath('Bal/Tp//Cd[contains(text(), "PRCD")]').first.ancestors('Bal')
+          bal = @xml_data.xpath('Bal/Tp//Cd[contains(text(), "OPBD") or contains(text(), "PRCD")]').first.ancestors('Bal')
```

Comment:
OPBD – Book balance of the account at the beginning of the account reporting period.
PRCD - Closing book balance of the previous day ([ref](http://www.finanssiala.fi/maksujenvalitys/dokumentit/ISO20022_Account_Statement_Guide_V1_3.pdf))